pkg/collectorinterface/const.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright © 2021 The MayaData Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collectorinterface
+
+const (
+	OpenebsCreateAnnotationSuffix = "event.openebs.io/volume-create"
```

Comment:
How about `VolumeCreateAnnotationSuffix`? Since it is not an entire event annotation name I want to convey it is a suffix part of annotation and it can have prefix as `nfs.<event.openebs.io/volume-create>`/`local.event.openebs.io/volume-create`.

Reply:
Updated
`OpenebsCreateAnnotationSuffix` --> `VolumeCreateEventAnnotation`  &
`OpenebsDeleteAnnotationSuffix` --> `VolumeDeleteEventAnnotation`